frontend/src/utilsLogic.tsx

```diff
@@ -216,7 +223,8 @@ export const getManagerInstances = (
     prettyName: spec?.prettyName,
     gui: guiEnabled,
     persistent: persistent,
-    idTemplate: name,
+    templateName: name,
+    templateId: makeTemplateKey(name, namespace.replace(/^workspace-/, '')),
```

Comment:
It should be better to get the workspace name from the labels. If this isn't possible it can be left like this...